src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1270,7 +1271,7 @@ static bool processContextAttributeVector (ContextElement*
                                       std::string("action: APPEND") +
                                       " - entity: [" + eP->toString() + "]" +
                                       " - offending attribute: " + targetAttr->toString() + 
-                                      " - attribute can not be appended");
+                                      " - attribute can not be appended");                
```

Comment:
These extra trailing spaces ... (it gives an error/warnings in the style-guide checker)